components/SearchTrackers.vue

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+    data () {
+      return {
+        /**
+         * example trackers list, where the value will be all relevant info for that tracker
+         * There is no need for these value lists to be equal size, some can have more/less pairs
+         */
+        trackers: {
+          "Alpha 01": { description: "Description of Alpha 01", nickname: "Florian" },
+          "Alpha 02": { description: "Description of Alpha 02", nickname: "Bo" },
+          "Alpha 03": { description: "Description of Alpha 03", nickname: "Ben" },
+          "Alpha 04": { description: "Description of Alpha 04", nickname: "Joshua" },
+          "Phone 01": { description: "Description of Phone 01", nickname: "Meg" },
+          "Phone 02": { description: "Description of Phone 02", nickname: "Jonathan", groups: "NCCN, flooding,  kitten stuck in a tree" }
+        }
+      };
+    },
+    mounted () {
+      /**
+       * event triggered on closing the details window, sent from the DetailsPane component
+       */
+      this.$root?.$on("details-closed", () => {
+        this.clearActive();
+      });
+    },
+    methods: {
+      /**
+       * function template in case we want to implement search
+       */
+        performSearch () {
+          console.log("performing search");
+        },
+        /**
+         * event triggered when clicking on a list item
+         * @param event the event that triggered the function
+         * @param details the list of details passed in, see the data()/trackers part
+         */
+        clickListItem (event : MouseEvent, details : Object) {
+          const targetElement = event.target as HTMLElement;
+          this.resetActive(targetElement);
+          // emit data through root, to be caught by the DetailsPane component
+          this.$root?.$emit("details", targetElement.innerText, details);
+        },
+        /**
+         * resets the active element to the given one, if it is not already active itself
+         * @param element the element for which to re-set the active class
+         */
+        resetActive (element : HTMLElement) {
+          const add : Boolean = !element.classList.contains("active");
+          this.clearActive();
+          if (add) {
+            element.classList.add("active");
+          }
+        },
+        /**
+         * gets called whenever the active element of the search list must be cleared
+         * and does this by looping over all 'active' elements
+         */
+        clearActive () {
+        const currentActive : HTMLCollectionOf<Element> = document.getElementsByClassName("active");
```

Comment:
The queryselector complained about the following linter issue:

> Property 'namedItem' is missing in type 'NodeListOf<Element>' but required in type 'HTMLCollectionOf<Element>'.Vetur(2741)
Using the getElementsByClassName seemed to work more easily, and since I'm used to it, I'd leave it here (since you also say in this case it is completely OK :) )